apps/examples/testcase/le_tc/kernel/tc_timer.c

```diff
@@ -273,7 +275,10 @@ static void tc_timer_timer_initialize(void)
 
 int timer_main(void)
 {
+#ifndef CONFIG_BUILD_PROTECTED
```

Comment:
Is CONFIG_BUILD_PROTECTED right? not APP_SEPARATION?

Reply:
CONFIG_BUILD_PROTECTED is fine.